VERSION

```diff
@@ -1 +1 @@
-v1.88.3-ske-2
+v1.88.3
```

Comment:
Entfernen wir das jetzt einfach weil eh egal? (Bauen ja nicht mehr über concourse)?

Reply:
genau, hab ich in der PR description auch nochmal explizit erwähnt.